### ISSUE TITLE
R-Api broken

### ISSUE THREAD

**Issue by PhilippPro** [openml/OpenML#748]
Tried to download datasets with the R API, but the following website does not work anymore, so I got an error: 
https://www.openml.org/api/v1/json/evaluation/list/tag/mlrRandomBot/function/area_under_roc_curve/limit/20000

Something changed?

Original R-Code: 
```
my_runs <- listOMLRunEvaluations(tag = "mlrRandomBot", limit = 20000, evaluation.measure = "area_under_roc_curve")
Downloading from 'https://www.openml.org/api/v1/json/evaluation/list/tag/mlrRandomBot/function/area_under_roc_curve/limit/20000' to '<mem>'.
Fehler in doHTTRCall(method, url = url, query = list(api_key = conf$apikey),  :
  ERROR (code = 546) in server response: Illegal filter`
```

**Comment by janvanrijn** [comment]
Sorry for taking so long to respond to this issue! 

* Error message was wrong. You ask for a limit of 20000 results, whereas 10000 is the max 

This, and another small fix are ready in #828 and waiting for review

**Comment by mfeurer** [comment]
Assuming that this is fixed as there was a PR merged for it -> closing it. Please reopen if the issue still exists.
